src/api/todos.ts

```diff
@@ -1,10 +1,18 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const USER_ID = 0;
+export const USER_ID = 3626;
+
+const todoUrl = (id?: number) => `/todos${id ? `/${id}` : ''}`;
```

Comment:
functions should be named so they indicate the action


```suggestion
const getTodoUrl = (id?: number) => `/todos${id ? `/${id}` : ''}`;
```